### PR TITLE
fix: historical comments issues

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**
fix issues according to https://github.com/apache/apisix-dashboard/pulls?q=is%3Apr+label%3A%22plugin+form%22+ comment
**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/components/Plugin/UI/cors.tsx [hunk 1]
```diff
@@ -57,9 +57,9 @@ const Cors: React.FC<Props> = ({ form }) => {
             <Select
               mode="multiple"
               optionLabelProp="label"
-              onChange={(value) => {
-                ((value as string[]).join(","));
-                if ((value as string[]).includes('*')) {
+              onChange={(value: string[]) => {
+                value.join(",")
```
Comment (juzhiyuan) [inline web/src/components/Plugin/UI/cors.tsx:61]: `join` will return a new value.

**Comment by juzhiyuan** [comment]
Please have a better title

File: web/src/components/Plugin/UI/cors.tsx (hunk 1 above)
Comment (juzhiyuan) [reply web/src/components/Plugin/UI/cors.tsx:61]: so why do we need this line?

**Review by liuxiran — APPROVED** [review]

File: web/src/components/Plugin/UI/cors.tsx (hunk 1 above)
Comment (LiteSun) [reply web/src/components/Plugin/UI/cors.tsx]: useless code. Just removed. cc @guoqqqi

**Review by juzhiyuan — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**